eiffel-vocabulary/EiffelActivityFinishedEvent.md

```diff
@@ -0,0 +1,34 @@
+# EiffelActivityFinishedEvent
+The EiffelActivityFinishedEvent declares that a previously queued activity (declared by [EiffelActivityQueuedEvent](./EiffelActivityQueuedEvent.md)) has finished.
+
+## Data Members
+### data.outcome
+__Type:__ Object  
+__Required:__ Yes  
+__Description:__ The outcome of the activity.
+
+#### data.outcome.verdict
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ SUCCESS, FAILURE, ERROR, ABORTED, TIMEOUT  
```

Comment:
I'm really struggling with this. I don't want to tie into Jenkins just for the sake of it (see below), and I think UNSTABLE is a really poor term. I'd much rather go with actual standards rather than de facto standards (and I'd argue that this isn't even that).

But this is an area where I'm having a hard time coming up with really good terminology. Arguably we should use FAULT, ERROR and FAILURE, but the way they're defined (see e.g http://programmers.stackexchange.com/questions/184412/whats-the-difference-between-fault-error-and-defect) I don't think helps in this case.

Regardless what we propose, we should add crystal clear explanations of each value in the description.